addons/website_slides/tests/test_ui_wslides.py

```diff
@@ -224,6 +224,10 @@ def _add_reaction(self, message, reaction):
             },
         )
 
+    def test_course_review_modification(self):
+        user_demo = self.user_demo
```

Comment:
It's best to test with the least possible access rights, in this case if possible testing as a portal user rather than as an internal user.